index.html

```diff
@@ -8,6 +8,38 @@
     <link rel="stylesheet" href="styles/shared.css" />
   </head>
   <body>
-    
+       <header class="container">
+              <div class="container-wrapper">
+                <div class="image">
+                  <img src="images/Illustration.png">
+                  <div class="description-text">
+                          <h5>HONEYMOON SPECİALS</h5>
+                          <h1>Our Romantic Tropical <br> Destinations</h1>
+                          <p>Et labore harum non nobis ipsum eum molestias mollitia et corporis praesentium a laudantium internos. Non quis eius quo eligendi corrupti et fugiat nulla qui soluta recusandae in maxime quasi aut ducimus illum aut optio quibusdam!</p>
+                          <button> <h6>View Packages </h6></button>
+                  </div>
+                <div class="plus-effects">
+                  <h3 class="hcolor">+</h3>
+                  <h3>+</h3>
+                  <h3>+</h3>
+                  <h3>+</h3>
+                  <h3>+</h3>

```

Comment:
bunları tek bir div içine grid veya flex kullanarak pozisyonlaman gerek. 'span' yada 'p' elementlerini kullanabilirsin h3 bunu oluşturmak için doğru bir element değil.